io/fv3atm_history_io.F90

```diff
@@ -445,7 +455,43 @@ subroutine history_type_output(hist, time, diag, atm_block, nx, ny, levs, ntcw,
                   ii = i + Atm_block%isc -1
                   nb = Atm_block%blkno(ii,jj)
                   ix = Atm_block%ixp(ii,jj)
-                  var2(i,j) = Diag(idx)%data(nb)%var2(ix)*lcnvfac
+                  if ( trim(diag(idx)%time_avg_kind) == 'rad_sw' .or. trim(diag(idx)%time_avg_kind) == 'rad_lw' .or. trim(diag(idx)%time_avg_kind) == 'rad_swlw_min') then
+                    select case (trim(diag(idx)%time_avg_kind))
+                      case ('rad_sw')
+                        this_radfh = time_radsw
+                      case ('rad_lw')
+                        this_radfh = time_radlw
+                      case ('rad_swlw_min')
+                        this_radfh = min(time_radsw,time_radlw)
+                      case default
+
+                    end select
+
+                    if (time_int == dt_atmos ) then
+                      var2(i,j) = Diag(idx)%data(nb)%var2(ix)*lcnvfac
+                      hist%rad_accum(idx,ii,jj) = Diag(idx)%data(nb)%var2(ix)
```

Comment:
The computation of accumulated radiation or physics stays in CCPP physics (fluxr), I suggest making the accumulation related change in ccpp, not in the IO file (fv3atm_history_io.F90). We only apply time scale or mask update in the fv3atm_history_io.F90.

Reply:
I looked the CCPP code, and see that the fluxr is accumulated every radiation hour (3600s) (multiplied by coszdg(i) / coszen(i)) in the code:
physics/physics/Interstitials/UFS_SCM_NEPTUNE/GFS_rrtmg_post.F90
so we need modify the bucket weighting in order to calculate the dswrf_ave and ulwrf correctly